release.sh

```diff
@@ -23,7 +23,14 @@ EXT=''
 if [ ${INPUT_GOOS} == 'windows' ]; then
   EXT='.exe'
 fi
-GOOS=${INPUT_GOOS} GOARCH=${INPUT_GOARCH} go build -o "${BINARY_NAME}${EXT}"
+
+GO_LDFLAGS=''
+if [ ! -z "${INPUT_LDFLAGS}" ]; then
+    GO_LDFLAGS="-ldflags \"${INPUT_LDFLAGS}\""
+fi
+
+
+bash -c "GOOS=${INPUT_GOOS} GOARCH=${INPUT_GOARCH} go build ${INPUT_BUILD_FLAGS} ${GO_LDFLAGS} -o ${BINARY_NAME}${EXT}"
```

Comment:
This extra `bash -c` seems useless, would you please remove it?

Reply:
It's not redundant without it, the command is not properly executed and since it handles GO_LDFLAGS as 3 strings, you can see the problem in this small script:

```
#!/bin/bash -x
INPUT_LDFLAGS="foo bar"
GO_LDFLAGS="-ldflags \"${INPUT_LDFLAGS}\""
go build ${GO_LDFLAGS} .
```

Reply:
OK. Then I'm ok with it. I'll try to solve it later.

Reply:
@mcuadros  I have solved it in https://github.com/wangyoucao577/go-release-action/pull/7 and released `v1.4`. It's ok to keep use `master` or maybe you want to use `v1.4`. It should not affect your using. If any error occurred, please let me know. Thanks!